cli/src/tests.rs

```diff
@@ -558,6 +558,43 @@ async fn create_workspace(tmp_path: TmpPath) {
         ));
 }
 
+#[rstest::rstest]
+#[tokio::test]
+async fn list_users(tmp_path: TmpPath) {
+    let tmp_path_str = tmp_path.to_str().unwrap();
+    let config = get_testenv_config();
+    let (url, [alice, ..], _) = run_local_organization(&tmp_path, None, config)
+        .await
+        .unwrap();
+
+    set_env(&tmp_path_str, &url);
+
+    Command::cargo_bin("parsec_cli")
+        .unwrap()
+        .args(["list-users", "--device", &alice.slughash()])
+        .assert()
+        .stdout(predicates::str::contains(format!("No users found",)));
+
+    Command::cargo_bin("parsec_cli")
+        .unwrap()
+        .args([
+            "create-workspace",
+            "--device",
+            &alice.slughash(),
+            "--name",
+            "new-workspace",
+        ])
+        .unwrap();
+
+    Command::cargo_bin("parsec_cli")
+        .unwrap()
+        .args(["list-users", "--device", &alice.slughash()])
+        .assert()
+        .stdout(predicates::str::contains(format!(
+            "Found {GREEN}3{RESET} user(s)",
+        )));
```

Comment:
Could you check the user list ?

Reply:
I can't know the users id

Reply:
Is it possible to only check the username (handle) ?

Reply:
I prefer to know that there are all users :thinking: